src/common.rs

```diff
@@ -23,7 +23,7 @@ pub static KEY: &str = "secret";
 pub static WORK_DIR: &str = "/tmp";
 
 // Secure mount of tpmfs (False is generally used for development environments)
-pub static MOUNT_SECURE: bool = true;
+pub static MOUNT_SECURE: bool = false;
```

Comment:
Yeah, I'm not sure - my somewhat narrow goal has been to get the code into a state that it can run with `docker-compose up` without encountering any errors. In theory this would eventually be an integration test or example that someone who clones the repo could just run in a snap to make sure things are working and see how it should behave. Is there a better way to do this? Maybe a development "feature" in Cargo.toml?

Reply:
I believe it results in the mount being in /tmp rather than a tmpfs mount. I figure if it helps Lily get moving for now we can we could keep it that way, but we should mark a todo or an issue. The other option is a config value to make it togglable outside of the code.

Reply:
I've changed this so that the flag is updated based on a features we can pass in: `cargo build --features=testing`.